bridges/snowbridge/primitives/router/src/inbound/mod.rs

```diff
@@ -104,7 +104,7 @@ pub struct MessageToXcm<
 }
 
 /// Reason why a message conversion failed.
-#[derive(Copy, Clone, TypeInfo, PalletError, Encode, Decode, RuntimeDebug)]
+#[derive(Copy, Clone, TypeInfo, PalletError, Encode, Decode, RuntimeDebug, PartialEq)]
```

Comment:
Why not also derive `PartialEq` when std is enabled like in https://github.com/Snowfork/polkadot-sdk/pull/119/files#diff-832917c2fd6302a9f175210f7d9941ea82a1a7c5a35aa35e662380074d405203R167?